### DISCUSSION TITLE
MQTTNet in Xamarain Form UWP

### DISCUSSION THREAD

**Discussion by ieie1986** [dotnet/MQTTnet#1337] [General]
I have already run android an ios application sucessfully using Xamarin Form. When I tried to test UWP application, I can not get data from mosquitto broker (MQTTnet IsConnected property is true).

Here is my capabilities manifest :
 <Capability Name="internetClientServer"/>

Is there something I missed?

**Comment by israelz330** [comment]
I cannot see your manifest. Have you tried enabling the Private Networks capability?
